### PR TITLE
Add select parameter in GET /contacts

### PR DESCRIPTION
**Pull request by Hugo-Prossaird**
This PR adds documentation for the select[] parameter in the ```GET /contacts endpoint```, which was previously missing. The select[] parameter allows clients to specify which fields to return.

### CODE REVIEW

File: docs/rest_api/contacts.rst [hunk 1]
```diff
@@ -246,6 +246,8 @@ Get a list of Contacts.
      - An array of advanced where conditions
    * - ``order``
      - An array of advanced order statements
+   * - ``select``
+     - An array of fields to include in the lead data, ID is required. If not provided, all fields are returned.
```
Comment (npracht) [inline docs/rest_api/contacts.rst]: I'd say "to field aliases" to be sure to properly understand.

**Review by npracht — APPROVED** [review]
After edition, good to go.

File: docs/rest_api/contacts.rst [hunk 2]
```diff
@@ -246,6 +246,8 @@ Get a list of Contacts.
      - An array of advanced where conditions
    * - ``order``
      - An array of advanced order statements
+   * - ``select``
+     - An array of field aliases to include in the lead data, ID is required. If not provided, all fields are returned.
```
Comment (RCheesley) [inline docs/rest_api/contacts.rst]:
```suggestion
     - An array of field aliases to include in the Contact data, with ID a mandatory field. Mautic returns all fields if it's not provided.
```

I think this should be Contact, or are we referring here to some legacy code where it still refers to lead? Also fixed the Vale warning about passive voice.

**Review by RCheesley — CHANGES_REQUESTED** [review]
Thanks for the PR @Hugo-Prossaird - some small tweaks to fix the grammar and a query on the terminology.

File: docs/rest_api/contacts.rst (hunk 2 above)
Comment (Hugo-Prossaird) [reply docs/rest_api/contacts.rst]: Thank you for your response, it has now been updated.

**Review by RCheesley — APPROVED** [review]
GTG Thanks @Hugo-Prossaird 🚀

**Pull request merged**